content/en/docs/concepts/workloads/controllers/garbage-collection.md

```diff
@@ -59,11 +59,22 @@ metadata:
 ```
 
 {{< note >}}
-Cross-namespace owner references are disallowed by design. This means:
-1) Namespace-scoped dependents can only specify owners in the same namespace,
-and owners that are cluster-scoped.
-2) Cluster-scoped dependents can only specify cluster-scoped owners, but not
-namespace-scoped owners.
+Cross-namespace owner references are disallowed by design.
```

Comment:
nit: What is an "owner reference"?

Reply:
ownerReferences are described in the paragraphs above

Reply:
OK. It may help to break down the two cases (if I understand correctly). Does line 64 apply to the following paragraph or both pargraphs?
Is this true:
Dependent objects are only owned (referenced) by objects within their namespace or cluster scope.

There are two types of scoping that affect object ownership and garbage collection:

1. Namespaced
Namespaced owners **must** exist in the same namespace as the dependent.
...
1. Cluster-scoped
Cluster-scoped dependents can only specify cluster-scoped owners.
...

Reply:
Hi @liggitt after this comment is addressed, I think this would be good to merge. PTAL

Reply:
Hi. I think the text is okay to merge. In the future, the content could be updated --  make the text easier to digest and localize.
/lgtm